velox/dwio/common/compression/PagedInputStream.h

```diff
@@ -28,7 +28,9 @@ class PagedInputStream : public dwio::common::SeekableInputStream {
       memory::MemoryPool& memPool,
       std::unique_ptr<Decompressor> decompressor,
       const dwio::common::encryption::Decrypter* decrypter,
-      const std::string& streamDebugInfo)
+      const std::string& streamDebugInfo,
+      bool useRawDecompression = false,
```

Comment:
Can we move streamDebugInfo to the last input?

Reply:
It has default arguments, streamDebugInfo can't be moved to last.